tests/phpunit/tests/themes/test-twenty-fourteen.php

```diff
@@ -42,52 +33,62 @@ public static function wpTearDownAfterClass() {
 		switch_theme( self::$stylesheet );
 	}
 
-	public function tear_down() {
-		parent::tear_down();
-
-		unset( $GLOBALS['polylang'] );
-	}
-
 	public function test_ephemera_widget() {
 		global $content_width; // The widget accesses this global, no matter what it contains.
 		$GLOBALS['wp_rewrite']->set_permalink_structure( '' );
 
-		$en = self::factory()->post->create( array( 'post_content' => 'Test', 'post_author' => 1 ) );
-		set_post_format( $en, 'aside' );
-		self::$model->post->set_language( $en, 'en' );
-
-		$fr = self::factory()->post->create( array( 'post_content' => 'Essai', 'post_author' => 1 ) );
-		set_post_format( $fr, 'aside' );
-		self::$model->post->set_language( $fr, 'fr' );
+		$posts = self::factory()->post->create_translated(
+			array(
+				'post_content' => 'Test',
+				'lang'         => 'en',
+			),
+			array(
+				'post_content' => 'Essai',
+				'lang'         => 'fr',
+			)
+		);
+		set_post_format( $posts['en'], 'aside' );
+		set_post_format( $posts['fr'], 'aside' );
 
 		$this->frontend->curlang = self::$model->get_language( 'fr' );
 
 		require_once get_template_directory() . '/inc/widgets.php';
-		$widget = new Twenty_Fourteen_Ephemera_Widget();
-		$args = array( 'before_title' => '', 'after_title' => '', 'before_widget' => '', 'after_widget' => '' );
+		$widget   = new Twenty_Fourteen_Ephemera_Widget();
+		$args     = array(
+			'before_title'  => '',
+			'after_title'   => '',
+			'before_widget' => '',
+			'after_widget'  => '',
+		);
 		$instance = array();
 
 		ob_start();
 		$widget->widget( $args, $instance );
 		$out = ob_get_clean();
 
-		$this->assertFalse( strpos( $out, '<p>Test</p>' ) );
-		$this->assertNotFalse( strpos( $out, '<p>Essai</p>' ) );
+		$this->assertStringNotContainsString( '<p>Test</p>', $out );
+		$this->assertStringContainsString( '<p>Essai</p>', $out );
 
 		unset( $content_width );
 	}
 
 	protected function setup_featured_tags() {
-		self::$tag_en = $en = self::factory()->term->create( array( 'taxonomy' => 'post_tag', 'name' => 'featured' ) );
-		self::$model->term->set_language( $en, 'en' );
-
-		self::$tag_fr = $fr = self::factory()->term->create( array( 'taxonomy' => 'post_tag', 'name' => 'en avant' ) );
-		self::$model->term->set_language( $fr, 'fr' );
-		self::$model->term->save_translations( $en, compact( 'en', 'fr' ) );
+		self::$tags = self::factory()->term->create_translated(
+			array(
+				'taxonomy' => 'post_tag',
+				'name'     => 'featured',
+				'lang'     => 'en',
+			),
+			array(
+				'taxonomy' => 'post_tag',
+				'name'     => 'en avant',
+				'lang'     => 'fr',
+			)
+		);
```

Comment:
You can leverage dedicated factory  and use `tag` property directly.

```suggestion
		self::$tags = self::factory()->tag->create_translated(
			array(
				'name'     => 'featured',
				'lang'     => 'en',
			),
			array(
				'name'     => 'en avant',
				'lang'     => 'fr',
			)
		);
```